torchvision/models/feature_extraction.py

```diff
@@ -497,7 +504,7 @@ def to_strdict(n) -> Dict[str, str]:
         for n in reversed(graph_module.graph.nodes):
             if n.op == "output":
                 orig_output_nodes.append(n)
-        assert len(orig_output_nodes)
+
```

Comment:
are we sure we want to remove this?

Reply:
I wanted to replace it with exception , but missed it.